opcua/client/ua_client.py

```diff
@@ -477,3 +477,14 @@ def history_read(self, params):
         self.logger.debug(response)
         response.ResponseHeader.ServiceResult.check()
         return response.Results
+
+
+def modify_monitored_item_low(self, params):
```

Comment:
Why  _low ? Low level? All methods have the same level here. Otherwise looks good.